middleman-breadcrumbs.gemspec

```diff
@@ -1,30 +1,30 @@
 $:.push File.expand_path("../lib", __FILE__)
-require "middleman-breadcrumbs/version"
+require 'middleman-breadcrumbs/version'
 
 Gem::Specification.new do |s|
-  s.name        = "middleman-breadcrumbs"
+  s.name        = 'middleman-breadcrumbs'
   s.version     = BreadcrumbsVersion::VERSION
   s.platform    = Gem::Platform::RUBY
-  s.authors     = ["Marnen Laibow-Koser"]
-  s.email       = ["marnen@marnen.org"]
-  s.homepage    = "http://github.com/marnen/middleman-breadcrumbs"
+  s.authors     = ['Marnen Laibow-Koser']
+  s.email       = ['marnen@marnen.org']
+  s.homepage    = 'http://github.com/marnen/middleman-breadcrumbs'
   s.summary     = %q{Breadcrumbs helper for Middleman}
   s.description = %q{Breadcrumbs helper for Middleman}
   s.license = 'MIT'
 
   s.required_ruby_version = '>= 2.0'
 
-  s.add_runtime_dependency "middleman", '>= 4.0.0'
+  s.add_runtime_dependency 'middleman', '>= 4.0.0'
   [
     'byebug',
     ['guard', '>= 2.10.5'],
     'guard-minitest',
     ['faker', '~> 1.5.0'],
     'rake'
-  ].each {|gem| s.add_development_dependency *gem }
+  ].each { |gem| s.add_development_dependency(*gem) }
```

Comment:
There's no reason for the parentheses here, and I prefer no space between `{` and `|`.